practices/managing-node-projects.md

```diff
@@ -10,7 +12,30 @@
 * [Dependency management tools](#dependency-management-tools)
 * [Handling dependencies from third parties](#handling-dependencies-from-third-parties)
 
-The following guide describes how we manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+The following guide describes how we use node and manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+
+
+## Specifying versions of node
+
+It is important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
+
+1. The [`engines`](https://docs.npmjs.com/files/package.json#engines) field in `package.json`:
+    * Important if authoring libraries, because `npm install` only _enforces_ compatibility on the basis of the `engines` field if the package is a dependency.
+    * Some deployment environments also respect it.
+    * It makes compatibility requirements explicit to developers working on your application.
+1. Using an `.nvmrc` file: An `.nvmrc` file is a configuration file for `nvm` (Node Version Manager). `nvm` enables developers to use different versions of node for different projects, on a per-directory basis. Projects should include an [`.nvmrc` file](https://github.com/creationix/nvm#nvmrc) in the root directory of the project to specify which version(s) of node are compatible. Developers can then run `nvm use` to use the right version of node, and [if using ZSH this shell script will run it automatically](https://github.com/creationix/nvm#calling-nvm-use-automatically-in-a-directory-with-a-nvmrc-file). Additionally [Travis respects `.nvmrc` files](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc), so using one will simplify your Travis configuration.
+
+### Run `nvm use` before `npm install`
+
+Developers should always ensure they are using the correct version of node (and implicitly `npm`) before doing an `npm install`.
+
+Running `nvm use` before `npm install` is good because:
+1. It make installs _more_ predictable, which minimises "but it works on my machine" issues.
+1. It minimises changes to the `package-lock.json` (if your project is comitting that file).
+
+A problem with using `npm` at the time of writing (September 2018) is that `npm install` does not guarantee reproducible builds, as neither the `package.json` nor `package-lock.json` is a source of authority for what is installed. The `npm ci` command will always install predictably (using the `package-lock.json` as a source of authority) but `npm ci` is not available in any current LTS version of node. It should be available with node 10 which should be in LTS [in October](https://nodejs.org/en/blog/release/v10.0.0/).
```

Comment:
contractions: "does not guarantee", "what is installed"

(unless you're using the expanded grammar for emphasis, then you can mark 'em up as emphasis)